src/sentry/loader/browsersdkversion.py

```diff
@@ -30,14 +30,14 @@ def load_registry(path):
 def get_highest_browser_sdk_version(versions):
     full_versions = [x for x in versions if _version_regexp.match(x)]
     return (
-        str(max(map(Version, full_versions)))
+        max(map(Version, full_versions))
```

Comment:
why did we remove the string cast here?

Reply:
I changed this file to operate on `Version` instances instead of `str`s, because I think we want to be able to do a simple version comparison [here](https://github.com/getsentry/sentry/pull/35310/files#diff-8fb4a3389f720525958f236f30b8e555b8a5975a6a1510542f68628b36a15495R33). This is also why a bunch of the tests had to be adjusted.